spring-kafka-streams/build.gradle

```diff
@@ -0,0 +1,19 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot'
+	id 'org.springframework.aot.smoke-test'
+	id 'org.graalvm.buildtools.native'
+}
+
+dependencies {
+	implementation(platform(org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES))
+	implementation("org.springframework.boot:spring-boot-starter")
+	implementation("org.springframework.kafka:spring-kafka:3.0.0-XAOT")
```

Comment:
What is `XAOT`?

Reply:
Oops - left over from when I was testing from my `mavenLocal()` before Andy suggested using `--include-build`.